engine/src/main/java/org/terasology/engine/world/chunks/remoteChunkProvider/ReceivedChunkInitialProvider.java

```diff
@@ -0,0 +1,35 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.engine.world.chunks.remoteChunkProvider;
+
+import org.joml.Vector3ic;
+import org.terasology.engine.world.chunks.Chunk;
+import org.terasology.engine.world.chunks.pipeline.InitialChunkProvider;
+
+import java.util.Comparator;
+import java.util.Set;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.PriorityBlockingQueue;
+
+public class ReceivedChunkInitialProvider implements InitialChunkProvider {
```

Comment:
I'd appreciate some Javadoc similar to the interface itself. What does "received" stand for? 

And should it be named 
```suggestion
public class ReceivedInitialChunkProvider implements InitialChunkProvider {
```

Reply:
Does the javadoc added in the latest commit look good?